docs/nlu/choosing-a-pipeline.rst

```diff
@@ -222,90 +213,83 @@ a full list of components. For example, these two configurations are equivalent:
     - name: "EntitySynonymMapper"
     - name: "SklearnIntentClassifier"
 
-Below is a list of all the pre-configured pipeline templates.
+Below is a list of all the pre-configured pipeline templates with customization information.
 
-.. _section_pretrained_embeddings_spacy_pipeline:
+.. _section_supervised_embeddings_pipeline:
 
-pretrained_embeddings_spacy
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+supervised_embeddings
+~~~~~~~~~~~~~~~~~~~~~
 
-To use the ``pretrained_embeddings_spacy`` template:
+To train a Rasa model in your preferred language, define the
+``supervised_embeddings`` pipeline as your pipeline in your ``config.yml`` or other configuration file:
 
-.. literalinclude:: ../../sample_configs/config_pretrained_embeddings_spacy.yml
+.. literalinclude:: ../../sample_configs/config_supervised_embeddings.yml
     :language: yaml
 
-See :ref:`language-support` for possible values for ``language``. To use
-the components and configure them separately:
+The ``supervised_embeddings`` pipeline supports any language that can be tokenized.  By default it uses whitespace
+for tokenization. You can customize the setup of this pipeline by adding or changing components. Here are the default
+components that make up the ``supervised_embeddings`` pipeline:
 
 .. code-block:: yaml
 
     language: "en"
 
     pipeline:
-    - name: "SpacyNLP"
-    - name: "SpacyTokenizer"
-    - name: "SpacyFeaturizer"
+    - name: "WhitespaceTokenizer"
     - name: "RegexFeaturizer"
     - name: "CRFEntityExtractor"
     - name: "EntitySynonymMapper"
-    - name: "SklearnIntentClassifier"
-
-.. _section_supervised_embeddings_pipeline:
+    - name: "CountVectorsFeaturizer"
+    - name: "EmbeddingIntentClassifier"
 
-supervised_embeddings
-~~~~~~~~~~~~~~~~~~~~~
+So for example, if your chosen language is not whitespace-tokenized (words are not separated by spaces), you
+can replace the ``WhitespaceTokenizer`` with your own tokenizer. We support a number of different :ref:`tokenizers <tokenizers>`,
+or you can :ref:`create your own <custom-nlu-components>`.
 
-To use it as a template:
+.. _section_pretrained_embeddings_spacy_pipeline:
 
-.. code-block:: yaml
+pretrained_embeddings_spacy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    language: "en"
+To use the ``pretrained_embeddings_spacy`` template:
 
-    pipeline: "supervised_embeddings"
+.. literalinclude:: ../../sample_configs/config_pretrained_embeddings_spacy.yml
+    :language: yaml
 
-The supervised embeddings pipeline supports any language that can be tokenized. The
-default is to use a simple whitespace tokenizer:
+See :ref:`pretrained-word-vectors` for supported values for ``language`` and instructions
+on how to link your own. To use the components and configure them separately:
```

Comment:
hm, but we don't actually list the values for language on the "language support" page anymore, so I don't think this sentence makes sense?

Reply:
```suggestion
To use the components and configure them separately:
```